### ISSUE TITLE
Cannot find python path

### ISSUE THREAD

**Issue by Neathle** [apommel/vscode-matlab-interactive-terminal#14]
Python is not installed or its path is incorrectly specified,
I have the correct path to python, i can call it from a comand prompt. Furthermore, if I specify the path, the program copies it without putting quotes around it, so the comand prompt cannot execute it, as python is installed in program files. I tried adding the quotes manually but that gives the same error.

**Comment by apommel** [comment]
Concerning the spaces in the path, that cam indeed cause issues and is not easy to fix with the current implementation. However, I am working on a partial rewrite that should fix this issue among others.
However, if python is in path, it should work. Is it installed from anaconda distribution? Do you have an alias for python?

**Comment by Neathle** [comment]
It is a fresh python installation, after reinstalling in C:\Python37 and explicitly setting the path in the extension settings, everything works, but even with python in PATH the extension is unable to find it

**Comment by apommel** [comment]
And you said it worked correctly if calling Python from a terminal? That's weird. I am glad you solved the issue, but I will try to find the cause of your problem as it should not happen. Thanks for your feedback.

**Comment by weypro** [comment]
I have the same problem. Anaconda for python 3.7.

**Comment by Jieli12** [comment]
me too +1

**Comment by Gallium70** [comment]
Install python into a directory WITHOUT whitespaces, and the Python Path should be set as "some/path/with/no/spaces/python.exe" . Otherwise neither putting the python path into quotes nor having python in PATH will help.

**Comment by alexanderVprice** [comment]
changing the path from default (empty field) to "py" worked for me

**Comment by TheBrenny** [comment]
Hitting this error too, but it's 100% not because of a bad python installation.

I did some testing to see what was going on and stepped through the code of the extension. The first thing I noticed was that any error with running a particular script was being defaulted to a "bad installation reason" ([see here](https://github.com/apommel/vscode-matlab-interactive-terminal/blob/53dfa710ea76fd804a6253a04209bf6ecb6ebbcb/src/extension.ts#L64-L67)).

So I decided to run the commands in my terminal to see what is wrong, and indeed, it's not because my python is bad, but because the python files are bad (or some snakes don't know how to read particular characters...). Here's the output:
```plain
  File "check_dependencies.py", line 1
SyntaxError: Non-ASCII character '\xc3' in file check_dependencies.py on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```
Which stems from the character `é` which is placed in the interface files [like this one](https://github.com/apommel/vscode-matlab-interactive-terminal/blob/53dfa710ea76fd804a6253a04209bf6ecb6ebbcb/interfaces/standard/check_dependencies.py#L1).

Weird that Python still tries to interpret comments, but it is what it is I guess. Fixes inbound.🙃
